test/FunctionalTests/MvcTests/TagHelperSampleTest.cs

```diff
@@ -47,7 +47,7 @@ public TagHelperSampleTest(SampleTestFixture<TagHelperSample.Web.Startup> fixtur
             AssertRedirectsToHome(createBobbyResponse);
 
             var expectedMediaType = MediaTypeHeaderValue.Parse("text/html; charset=utf-8");
-            var resourceFile = "FunctionalTests.resources.TagHelperSample.Web.Home.Index.html";
+            var resourceFile = "TagHelperSample.Web.Home.Index.html";
```

Comment:
Why was a change necessary here? Appear to be calculating the same name that was previously provided.